releasenotes/notes/no-dateutil-ca2ca5f5a75fbb22.yaml

```
@@ -0,0 +1,10 @@
+---
+upgrade:
+  - |
+    The ``python-dateutil`` library is no longer a dependency of Qiskit. Since
+    Qiskit 2.0.0 nothing in the library wa actively using the library anymore
```

A small typo:

```suggestion
    Qiskit 2.0.0 nothing in the library was actively using the library anymore
```

You are also using the word "anymore" a bit too many times in one short paragraph. 😛

---

I can try to avoid using the word anymore, but I'm not sure the release note would be understandable anymore, it's hard to tell if nobody is reading the word anymore anymore.

---

Lol. Fair enough, I will not waste your time with the word "anymore" anymore. 😄